axum-extra/src/middleware.rs

```diff
@@ -30,6 +30,7 @@ use tower_layer::Identity;
 /// `axum_extra::middleware::option_layer` makes sure that the output `Body` is [`axum::body::Body`].
 ///
 /// [`Layer`]: tower_layer::Layer
+/// [`tower::util::option_layer`]: https://docs.rs/tower/latest/tower/util/fn.option_layer.html
```

Comment:
Instead if the http link, can you make tower optional and put it behind the `__private_docs` feature?

Reply:
Yes. Since it isn't used in the code in the document, used the http link, but it can be `__private_doc` feature as well. Updated to use it.

Reply:
Yes. Since it isn't used in the document code block, used the http link, but it can be the `__private_doc` feature as well. Updated to use it.